kuma/settings/common.py

```diff
@@ -1757,11 +1757,6 @@ def get_user_url(user):
     "curl",
 ]
 
-# TODO: Once using DRF more we need to make that exception handler more generic
-REST_FRAMEWORK = {
-    'EXCEPTION_HANDLER': 'kuma.search.utils.search_exception_handler'
```

Comment:
So weird that the error handling was attached to the `search` app when it should be generic. A relic from the past.